db/seeds.d/050-taxonomies.rb

```diff
@@ -8,7 +8,11 @@
     taxonomy.without_auditing do
       tax_name = ENV.fetch("SEED_#{taxonomy.to_s.upcase}", "Default #{taxonomy}")
       tax = taxonomy.create!(name: tax_name)
-      associations = taxonomy.reflect_on_all_associations.reject {|assoc| skip_associations.include?(assoc.name)}
+      associations = associations.select do |assoc|
+        !skip_associations.include?(assoc.name) &&
+        (assoc.is_a?(ActiveRecord::Reflection::HasManyReflection) ||
+            assoc.is_a?(ActiveRecord::Reflection::HasAndBelongsToManyReflection))
+      end
```

Comment:
I thought about just making sure this isn't a HasOne, but that wouldn't catch a BelongsTo (which probably doesn't exist, but may).  Also, this seems to be assigning both the HasMany and HasMany Through relationships, which seemed needless.

Reply:
most associations are using `:through => :taxable_taxonomies`, so if we skip the `ActiveRecord::Reflection::ThroughReflection` ones we won't assign any of them. I'd rather we skip `HasOne` and `BelongsTo` specifically.